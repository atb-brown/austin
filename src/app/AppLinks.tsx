import PortfolioLink from "../component/portfoliolink/PortfolioLink";
import React from "react";

/**
 * The primary list of links that I want prominently displayed on my site.
 *
 * @return {ReactElement}
 */
export default function AppLinks(): JSX.Element {
  const linksHdr = "Links";

  return (
    <div
      style={{
        alignItems: "center",
        display: "flex",
        flex: 1,
        flexDirection: "column",
      }}
    >
      <h1>{linksHdr}</h1>
      <PortfolioLink
        description="GitHub"
        url="https://github.com/atb-brown"
        urlId="atb-brown"
      />
      <PortfolioLink
        description="StackOverflow"
        url="https://stackoverflow.com/users/5476186/austin-brown"
        urlId="5476186/austin-brown"
      />
      <PortfolioLink
        description="LinkedIn"
        url="https://www.linkedin.com/in/austin-brown-06382a12a/"
        urlId="austin-brown-06382a12a"
      />
      <PortfolioLink
        description="Scrum Master Certification"
        issuerDescription="Scrum Alliance, Inc."
        issuerUrl="https://www.scrumalliance.org/"
        url="https://bcert.me/sixkczuof"
        urlId="sixkczuof"
      />
      <PortfolioLink
        description="SAFe 5 Scrum Master Certification"
        issuerDescription="Scaled Agile, Inc."
        issuerUrl="https://scaledagile.com/"
        url="https://www.credly.com/badges/18e3c923-956c-4525-b857-5f9c0dfb9713/public_url"
        urlId="18e3c923-956c-4525-b857-5f9c0dfb9713"
      />
    </div>
  );
}
