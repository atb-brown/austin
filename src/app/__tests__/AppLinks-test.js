import AppLinks from "../AppLinks";
import renderer from "react-test-renderer";

it("renders to match snapshot", () => {
  const tree = renderer.create(<AppLinks />);

  expect(tree).toMatchSnapshot();
});
