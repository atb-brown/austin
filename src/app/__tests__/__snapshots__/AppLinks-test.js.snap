// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match snapshot 1`] = `
<div>
  <h1>
    Links
  </h1>
  <div
    className="App-text"
  >
    <div>
      GitHub
    </div>
    <div>
      <a
        className="App-link"
        href="https://github.com/atb-brown"
      >
        atb-brown
      </a>
    </div>
    <br />
  </div>
  <div
    className="App-text"
  >
    <div>
      StackOverflow
    </div>
    <div>
      <a
        className="App-link"
        href="https://stackoverflow.com/users/5476186/austin-brown"
      >
        5476186/austin-brown
      </a>
    </div>
    <br />
  </div>
  <div
    className="App-text"
  >
    <div>
      LinkedIn
    </div>
    <div>
      <a
        className="App-link"
        href="https://www.linkedin.com/in/austin-brown-06382a12a/"
      >
        austin-brown-06382a12a
      </a>
    </div>
    <br />
  </div>
  <div
    className="App-text"
  >
    <div>
      Scrum Master Certification
    </div>
    <div>
      <a
        className="App-link"
        href="https://bcert.me/sixkczuof"
      >
        sixkczuof
      </a>
    </div>
    <div
      className="App-soft-text"
    >
      Issued by: 
      <a
        className="App-link"
        href="https://www.scrumalliance.org/"
      >
        Scrum Alliance, Inc.
      </a>
    </div>
    <br />
  </div>
  <div
    className="App-text"
  >
    <div>
      SAFe 5 Scrum Master Certification
    </div>
    <div>
      <a
        className="App-link"
        href="https://www.credly.com/badges/18e3c923-956c-4525-b857-5f9c0dfb9713/public_url"
      >
        18e3c923-956c-4525-b857-5f9c0dfb9713
      </a>
    </div>
    <div
      className="App-soft-text"
    >
      Issued by: 
      <a
        className="App-link"
        href="https://scaledagile.com/"
      >
        Scaled Agile, Inc.
      </a>
    </div>
    <br />
  </div>
</div>
`;
